internal/uvm/create.go

```diff
@@ -273,6 +273,15 @@ func (uvm *UtilityVM) Close() (err error) {
 	if uvm.hcsSystem != nil {
 		return uvm.hcsSystem.Close()
```

Comment:
If we return here we won't run the code below.

Reply:
yeah, good catch. realized that too while testing.